Ch hound ci with badges 184637123

nkurunziza1 opened the pull request:
**What does this PR do?**
• Fixed HoundCI badge in the Readme file

**Description**

This pull request adds HoundCI badges to the repository, which will display the results of automated linting checks on all JavaScript files using ESLint.  The linter should be run on all files with the ".js" or ".jsx" extension and uses a configuration file ".eslintrc.json". This will help to ensure that the code meets the specified standards and best practices.

mjeanbosco19 reviewed (changes requested):
1. I noticed that you have 8 commits, I recommend you To "squash" in Git means to combine your multiple commits into one commit or at least not above two commits.
2. Which issue did you face with? I noticed that you have changed every single quote to double quote which can cause conflict for those who have single quote?

Abdulkeza commented:
> 1. I noticed that you have 8 commits, I recommend you To "squash" in Git means to combine your multiple commits into one commit or at least not above two commits.
> 2. Which issue did you face with? I noticed that you have changed every single quote to double quote which can cause conflict for those who have single quote?

for a double quote, that means as a team, you are going to use the same structure. ie only double quote allowed.

mjeanbosco19 reviewed (changes requested):
**Great job @nkurunziza1!**
However, you should work on this.

1. Squash your 10 commits into one or at least two commits.
2. Remember to write description of your PR.

divinecharlotte committed:
chore(Integrate CircleCi and Coveralls badges ):fix coveralls badges

feat(register-user):
implement user registration with passport strategy and jwt

feat(register-user):
implement user registration with passport strategy and jwt

chore(postgresDB and sequelize):DB configuration

chore(Integrate CircleCi and Coveralls badges ):testing badges

chore(project setup): setup express boilerplate

adding eslint config

comment controler code

uncommenting the index code

removing double quotes in my code

add hound.json

adding hound badge on readme

checking of houndCi if it is working

adding support code quality on houndci

 commit about unstaged changes

adding eslint config

comment controler code

uncommenting the index code

removing double quotes in my code

adding hound badge on readme

checking of houndCi if it is working

adding support code quality on houndci

squash a commits

adding circleci badges

chore(Integrate CircleCi and Coveralls badges ):fix coveralls badges

remove double circleci badges

solve conflicts

nkurunziza1 committed:
Update README.md

remove doubled circleci coverage
